src/hivedscheduler/deploy/hivedscheduler.yaml.template

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hivedscheduler-sts
+  namespace: default
+spec:
+  serviceName: hivedscheduler
+  selector:
+    matchLabels:
+      app: hivedscheduler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: hivedscheduler
+    spec:
+      serviceAccountName: hivedscheduler-account
+      containers:
+      - name: defaultscheduler
+        image: gcr.io/google_containers/kube-scheduler:v1.14.2
+        command: [
+          "/usr/local/bin/kube-scheduler",
+          "--master={{ cluster_cfg['layout']['kubernetes']['api-servers-url'] }}",
+          "--config=/hivedscheduler-config/config.yaml",
+          "-v=4"]
+        volumeMounts:
+        - name: hivedscheduler-config
+          mountPath: /hivedscheduler-config
+      - name: hivedscheduler
+        image: hivedscheduler/hivedscheduler:v0.1.0
+        command: [
+          "bash", "-c",
+          "cp /hivedscheduler-config/hivedscheduler.yaml . &&
+          ./start.sh"]
+        env:
+          - name: KUBE_APISERVER_ADDRESS
+            value: "{{ cluster_cfg['layout']['kubernetes']['api-servers-url'] }}"
```

Comment:
Should check whether it's a rbac cluster

Reply:
disable this env still cannot run in aks cluster, needs more changes after tested